Classes/Queue/FileQueue.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pluswerk\Sentry\Queue;
+
+use Exception;
+use TYPO3\CMS\Core\Core\Environment;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class FileQueue implements QueueInterface
+{
+    private string $directory;
+    public function __construct()
+    {
+        $this->directory =  Environment::getVarPath() . '/tx_plussentry_queue/';
+        if (!file_exists($this->directory)) {
+            try {
+                GeneralUtility::mkdir_deep($this->directory);
+            } catch (Exception $exception) {}
+        }
+    }
+
+    public function pop(): ?Entry
```

Comment:
maybe we should not silence all the errors for the pop function?

This is the only possibility there we can find out if something is wrong and can safely report it to the Sentry Instance.

Reply:
done